subgraphs/portal/src/mappings/helpers.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Address,
+  BigInt,
+  Bytes,
+  json,
+  JSONValue,
+  log,
+} from "@graphprotocol/graph-ts";
+
+import { DestBySource, SourceByDest } from "../common/crossTokenAddress";
+
+import { ZERO_ADDRESS } from "../sdk/util/constants";
+
+export function getCrossTokenAddress(
```

Comment:
why not return `Bytes` instead since you convert it to Bytes every time you call this function?